README.md

```diff
@@ -39,10 +39,30 @@
 * SXSW
 * Bear claws
 * Pearl Jam
+* Freshcraft bar
 * HipChat
 * Trend IMSS
 * Joe Nacchio
 * Ubuntu
 * Complainy Hulu customers
 * Places called "Fish Tacos"
 * Going south of Alameda
+
+## Quotes
+* Wait a tic
+* Al, I'll miss you the most
+* I have the sys admin physique
+* Everything's coming up Milhouse
+* It's not "slash user", it's "slash U, S, R"
+* Cold food is for peasants
+* Subarus are so hot
+* Davita drivers are asshats!
+* Get off my lawn!
+* Let's go back. First, there were the dinosaurs
+* I hate java
+* Douche nozzle
+* Dude, stop cutting into our ESOP
+* Go f\*ck yourself!
+* F\*ck you, f\*ck you, you're cool, and f\*ck you
```

Comment:
this is technically a re-quote, but I guess it can slide.

Reply:
True. I'd like to keep this original. WIll remove.